src/admin.ts

```diff
@@ -57,111 +56,48 @@ export class Admin {
    *
    * @param command - The command to execute
    * @param options - Optional settings for the command
-   * @param callback - An optional callback, a Promise will be returned if none is provided
    */
-  command(command: Document): Promise<Document>;
-  command(command: Document, options: RunCommandOptions): Promise<Document>;
-  /** @deprecated Callbacks are deprecated and will be removed in the next major version. See [mongodb-legacy](https://github.com/mongodb-js/nodejs-mongodb-legacy) for migration assistance */
-  command(command: Document, callback: Callback<Document>): void;
-  /** @deprecated Callbacks are deprecated and will be removed in the next major version. See [mongodb-legacy](https://github.com/mongodb-js/nodejs-mongodb-legacy) for migration assistance */
-  command(command: Document, options: RunCommandOptions, callback: Callback<Document>): void;
-  command(
-    command: Document,
-    options?: RunCommandOptions | Callback<Document>,
-    callback?: Callback<Document>
-  ): Promise<Document> | void {
-    if (typeof options === 'function') (callback = options), (options = {});
-    options = Object.assign({ dbName: 'admin' }, options);
-
+  async command(command: Document, options?: RunCommandOptions): Promise<Document> {
     return executeOperation(
       this.s.db.s.client,
-      new RunCommandOperation(this.s.db, command, options),
-      callback
+      new RunCommandOperation(this.s.db, command, { dbName: 'admin', ...options })
```

Comment:
this isn't any different than the previous behavior, but since we aren't guarding against non-object options, all our APIs will have some odd behavior if passed a string here, which is different from any api that passes options directly to the RunCommandOperation which would fail at resolveBSONOptions (based on code analysis); something that we may want to standardize/improve at some point anyway

Reply:
Agreed, and something like verify something is an object is cheap, but taking it much futher could have diminishing returns. As long as there aren't known silent failures (unless there's runtime typeof checking) generally it feels fair to lean on our documentation / TS types to communicate correctness, and failures from breaking the TS "contract" are expected behavior.

As for this spot, we could do a little ternary `...(typeof options === 'object' ? options : {})` thoughts?

Reply:
I don't really care about this for this PR or v5 in general; validation on high traffic paths is not a good idea, just something to keep in mind, or add to the tech chat agenda for future improvement - making the failure more consistent in the case of bad options, preferably by creating a suitable place for it to happen "naturally"

Reply:
added a bullet to tech chat